dir.props

```diff
@@ -144,6 +144,12 @@
     -->
     <SkipBuildPackages>true</SkipBuildPackages>
   </PropertyGroup>
+  
+  <PropertyGroup>
+    <!-- By default make all libraries to be AnyCPU but individual projects can override it if they need to -->
+    <Platform>AnyCPU</Platform>
+    <OutputType>Library</OutputType>
+  </PropertyGroup>
```

Comment:
Should these then be removed from all of our other .csproj files that specify the same values (which I believe is most of them)?